### DISCUSSION TITLE
LabXP - Help with the tutorials

### DISCUSSION THREAD

**Discussion by EduBrancher** [kworkflow/kworkflow#466] [General]
Hi guys, I don't know if this is the best place to discuss this topic, but we are trying to follow the QEMU configuration tutorial in:

https://flusp.ime.usp.br/others/use-qemu-to-play-with-linux/

and we can't create the shared directory. It seems that homectl simply breaks everything and does not work with our settings (we installed Arch).

I was wondering if we could set up a meeting to ask questions about the tutorial?

Regards
Edu

**Comment by rodrigosiqueira** [comment]
Hi @EduBrancher could you translate this topic to English?

**Comment by EduBrancher** [comment]
> Hi guys, I don't know if this is the best place to discuss this topic, but we are trying to follow the QEMU configuration tutorial in:
> 
> https://flusp.ime.usp.br/others/use-qemu-to-play-with-linux/
> 
> and we can't create the shared directory. It seems that homectl simply breaks everything and does not work with our settings (we installed Arch).
> 
> I was wondering if we could set up a meeting to ask questions about the tutorial?
> 
> Regards Edu

EDIT: We managed to finish the tutorial. The issue was our samba configurations. It seems that in ARCH Linux, samba needs a configuration step before the steps listed in the tutorial can work. So perhaps it would be best for the tutorial to add this information for ARCH installations.

**Reply by kwy95** [reply]
That's very good! The FLUSP website is open source and the code can be found [here](https://gitlab.com/flusp/site). If you want to you can create the merge request to add these steps to the tutorial page.
